steward/src/commands/schedule_corks/aave_cellar/fees_distributor.rs

```diff
@@ -0,0 +1,59 @@
+use crate::{application::APP, prelude::*, utils::SubmitCork};
+use abscissa_core::{clap::Parser, Command, Runnable};
+use ethers::abi::AbiEncode;
+use steward_abi::aave_v2_stablecoin::*;
+
+/// Fees Distributor subcommand
+#[derive(Command, Debug, Parser)]
+#[clap(
+    long_about = "DESCRIPTION \n\n Set fees distributor of target Cellar.\n This command will set the fees distributor of a Cellar. This is a validator only command and can only be run by validators. It sets the fees distributor of a Cellar based on the height specified by \n the validators. Therefore, it'll execute the function when the chain reaches that height. This command also takes the fee distributor's address"
```

Comment:
It may be worth mentioning in this description that the fees distributor is a Cosmos address that the contract will bridge funds to.